lib/inspec/profile_vendor.rb

```diff
@@ -107,14 +107,20 @@ def warm_vendor_cache_from_archives
         # Place the local dependency in the vendor cache.
         # Fetchers are in charge of calculating cache keys.
         fetcher = Inspec::Fetcher::Local.resolve(dep_profile_path)
-        dep_profile_cache_dir = "#{cache_path}/#{fetcher.cache_key}"
-        file_provider.extract(dep_profile_cache_dir)
+        # Use a shorter name here in hopes of dodging windows filesystems path length restrictions
+        actual_dep_profile_cache_dir = "#{cache_path}/#{fetcher.cache_key}"
+        short_dep_profile_cache_dir = "#{cache_path}/#{SecureRandom.hex[0..6]}"
```

Comment:
This doesn't have to be a random name, does it? Literally `"#{cache_path}/short"` would work. If the concern is collision, then mktmpdir might be a better bet.